bibtex-completion.el

```diff
@@ -59,6 +59,48 @@
   "Helm plugin for searching entries in a BibTeX bibliography."
   :group 'completion)
 
+(defface bibtex-completion-author-face
+  '((t :inherit font-lock-keyword-face))
+  "Face for authors in `bibtex-completion-format-entry'."
+  :group 'bibtex-completion)
+
+(defface bibtex-completion-year-face
+  '((t :inherit font-lock-function-name-face))
+  "Face for bibtex entry year in `bibtex-completion-format-entry'."
+  :group 'bibtex-completion)
+
+(defface bibtex-completion-title-face
+  '((t :inherit default))
+  "Face for bibtex entry title in `bibtex-completion-format-entry'."
+  :group 'bibtex-completion)
+
+(defface bibtex-completion-type-face
+  '((t :inherit font-lock-string-face))
+  "Face for bibtex entry type in `bibtex-completion-format-entry'."
+  :group 'bibtex-completion)
+
+(defface bibtex-completion-pdf-face
+  '((t :inherit font-lock-type-face))
+  "Face for `bibtex-completion-pdf-symbol' when entry has a pdf."
+  :group 'bibtex-completion)
+
+(defface bibtex-completion-pdf-missing-face
```

Comment:
Is this to facilitate the icons as I do in `bibtex-actions`?

If yes, when I tested adding icons earlier the symbols, I couldn't get it to work. Have you?

Reply:
> Is this to facilitate the icons as I do in bibtex-actions?

Yes. If you look at the screenshot at the top of this PR it should be working.

> If yes, when I tested adding icons earlier the symbols, I couldn't get it to work. Have you?

What's the issue exactly? Did you set `bibtex-completion-format-always-show-symbols` to `t`? The default behaviour is to hide the symbols when the note/pdf is missing and I didn't want to break compatability with that so symbols are only always shown when that option is set to true (it's disabled by default).

Reply:
I don't remember what the issue was TBH. But glad it works.

On a related note, can you take a look at #366? I got stuck, not being able to figure out why it wasn't working for me.

And it relates to this PR ;-)